_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: drupal drupal8 drupal-planet marketing 
+summary: SEO trends 2017 and Drupal 8 as the best option for your long-term marketing campaign.
```

Comment:
@kostajh 2016 is over for marketers. Anything that is marketing related this late in the year should be focused on 2017 - that was intentional.